.changeset/gentle-chefs-smoke.md

```diff
@@ -0,0 +1,12 @@
+---
+"wrangler": patch
+---
+
+Colorize Deployed Bundle Size
+Most bundlers, and other tooling that give you size outputs will colorize their the text to indicate if the value is within certain ranges.
+The current range values are:
+red 100% - 90%
+yellow 89% - 70%
+green <70%
```

Comment:
It might make more sense to users to show these as Mb/Kb rather than a percentage?

Reply:
They don't see these values as output, its the percentage of the max allowed and current size. Since the max may change in the future I rather stick with percentage.